doc/source/tune/doc_code/faq.py

```diff
@@ -93,20 +93,6 @@ def train_func(config):
     tuner.fit()
     # __resources_pgf_end__
 
-    # __resources_scalingconfig_start__
```

Comment:
Disable using `ScalingConfig` in `Tune.with_resources`. Ensure that `ScalingConfig` is the class only for Ray Train.